iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/preprocessing/main/oozie_app/workflow.xml

```diff
@@ -119,6 +119,12 @@
 			<description>metadata extraction cache location, path pointing to
 				root cache directory holding meta.json file</description>
 		</property>
+		<property>
+            <name>referenceextraction_project_fundingclass_blacklist_regex</name>
+            <value>$^</value>
+            <description>regexp matching funding class, describes projects which 
+            should be blacklisted by reference extraction algorithm</description>
+        </property>
```

Comment:
I hate to be that guy ;) but indentation. Can you set it so the new code has the same way of indenting the code as the rest of it? Maybe there's an option for this in Eclipse?